### PR TITLE
docs(connectors): Add info when to use message start events as inboun…

### PR DESCRIPTION
**Pull request by sbuettner**
…d events.

## Description

https://github.com/camunda/team-connectors/issues/811

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [ ] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by sbuettner** [comment]
@christinaausley would be great if you could give this a second look and whether the text is understandable and not too technical

**Comment by akeller** [comment]
@sbuettner I noticed you didn't backport this change. Are you waiting for a review before backporting? Or is this not applicable to other versions?

**Comment by sbuettner** [comment]
Yes, exactly. I did not want to change it again all over the place :-)

**Comment by sbuettner** [comment]
Thanks @chillleader & @christinaausley. I added the docs to previous versions. Good to go from my side now.

File: docs/components/connectors/use-connectors/inbound.md [hunk 1]
```diff
@@ -12,7 +12,13 @@ Review our [list of existing inbound Connectors](/components/connectors/out-of-t
...
+Inbound Connectors are modeled as **catch events** in BPMN. Connectors that create a new process instance are modeled as **start events**, and Connectors that send messages to an already running process instance are modeled as **intermediate catch events**. It is also possible to create new processes by using a **message start events** and to use Inbound Connects via **boundary events**
+
+:::important
+If **idempotency** is a concern for the process creation and reprocessing of messages should never lead to a duplicate process instance creation, it is recommended to use the **message start event** element for an Inbound Connector as it relies on publishing a message.
+
+**Message start events** support the **Message ID** property that allows to specify a unique value that will be used by Zeebe to [guarantee uniqueness](/components/concepts/messages.md#message-uniqueness) in case other messages are published that use the same **Message ID**.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/connectors/use-connectors/inbound.md]: @sbuettner Is this an ongoing conversation, or are we planning to make this change?

[the remark this answers is not in the document]
Comment (sbuettner) [reply docs/components/connectors/use-connectors/inbound.md]: I thought it was potentially incorporated into your commit. Should we integrate the suggestion. I am indifferent so happy to get your perspective. Feel free to make the change if it reads better 😄

[the remark this answers is not in the document]
Comment (sbuettner) [reply docs/components/connectors/use-connectors/inbound.md]: @chillleader I incorporated your idea.

**Review by chillleader — APPROVED** [review]
Thanks for the changes!

**Pull request merged**